### PR TITLE
add new cases go_check_GUI_driver and update cases.xml for it

### PR DESCRIPTION
**Pull request by dulili**

### CODE REVIEW

File: testscripts/go_check_GUI_driver.ps1 [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+###############################################################################
+#
+# Main Body
+#
+###############################################################################
+
+$retVal = $Failed
+
+
```
Comment (yangboroy) [inline testscripts/go_check_GUI_driver.ps1]: No matter RHEL6, RHEL7 or RHEL8, they own this package, right?

File: testscripts/go_check_GUI_driver.ps1 [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+# Check the scsi timeout value in two files.
+$vmware_driver = bin\plink.exe -i ssh\${sshKey} root@${ipv4} "rpm -qa xorg-x11-drv-vmware"
+Write-Host -F Red "driver result is $vmware_driver"
```
Comment (yangboroy) [inline testscripts/go_check_GUI_driver.ps1]: Suggest: Add a Write-Output to log file about this VAR

File: testscripts/go_check_GUI_driver.ps1 [hunk 3]
```diff
@@ -0,0 +1,151 @@
+###############################################################################
+##
+## Description:
+## Check the vmware driver xorg-x11-drv-vmware exist in guest.
+##
+###############################################################################
+##
+## Revision:
+## V1.0 - ldu - 05/20/2019 - Check the vmware driver xorg-x11-drv-vmware exist in guest.
```
Comment (yangboroy) [inline testscripts/go_check_GUI_driver.ps1]: ##  v1.0.0 - ldu - 05/20/2019 - Create the script to check xorg-x11-drv-vmware

File: testscripts/go_check_GUI_driver.ps1 (hunk 1 above)
Comment (dulili) [reply testscripts/go_check_GUI_driver.ps1]: yes, if install GUI, this package should be installed.

File: testscripts/go_check_GUI_driver.ps1 (hunk 3 above)
Comment (dulili) [reply testscripts/go_check_GUI_driver.ps1]: OK

File: testscripts/go_check_GUI_driver.ps1 (hunk 2 above)
Comment (dulili) [reply testscripts/go_check_GUI_driver.ps1]: OK

**Pull request merged**